processor/src/chiplets/aux_trace/mod.rs

```diff
@@ -543,28 +544,18 @@ fn build_pipe_request<E: FieldElement<BaseField = Felt>>(
     req1 * req2
 }
 
-/// Builds `RCOMBBASE` requests made to the memory chiplet.
-fn build_rcomb_base_request<E: FieldElement<BaseField = Felt>>(
+/// Builds `HORNERBASE` or `HORNEREXT` requests made to the memory chiplet.
+fn build_horner_eval_request<E: FieldElement<BaseField = Felt>>(
     main_trace: &MainTrace,
     alphas: &[E],
     row: RowIndex,
 ) -> E {
-    let tz0 = main_trace.helper_register(0, row);
-    let tz1 = main_trace.helper_register(1, row);
-    let tzg0 = main_trace.helper_register(2, row);
-    let tzg1 = main_trace.helper_register(3, row);
-    let a0 = main_trace.helper_register(4, row);
-    let a1 = main_trace.helper_register(5, row);
-    let z_ptr = main_trace.stack_element(13, row);
-    let a_ptr = main_trace.stack_element(14, row);
+    let a0 = main_trace.helper_register(0, row);
+    let a1 = main_trace.helper_register(1, row);
+    let a_ptr = main_trace.stack_element(13, row);
     let op_label = MEMORY_READ_WORD_LABEL;
 
-    let req1 =
-        compute_mem_request_word(main_trace, op_label, alphas, row, z_ptr, [tz0, tz1, tzg0, tzg1]);
-    let req2 =
-        compute_mem_request_word(main_trace, op_label, alphas, row, a_ptr, [a0, a1, ZERO, ZERO]);
-
-    req1 * req2
+    compute_mem_request_word(main_trace, op_label, alphas, row, a_ptr, [a0, a1, ZERO, ZERO])
```

Comment:
I would use more descriptive names than `a*`

```suggestion
    let alpha_low = main_trace.helper_register(0, row);
    let alpha_high = main_trace.helper_register(1, row);
    let alpha_ptr = main_trace.stack_element(13, row);
    let op_label = MEMORY_READ_WORD_LABEL;

    compute_mem_request_word(main_trace, op_label, alphas, row, alpha_ptr, [alpha_low, alpha_high, ZERO, ZERO])
```

Reply:
The issue is that we are already using alpha for something else. I have now opted for `eval_point` as it is more descriptive and doesn't collide with `alphas`